src/Binance/Entry.cpp

```diff
@@ -24,7 +23,7 @@ std::string Entry::deriveAddress([[maybe_unused]] TWCoinType coin, const PublicK
 Data Entry::addressToData([[maybe_unused]] TWCoinType coin, const std::string& address) const {
     Address addr;
     if (!Address::decode(address, addr)) {
-        return Data();
+        return {};
```

Comment:
Can you explain the argument for `{}` instead of `Data()`?

Reply:
Data is already specified in the return type, with our current clang-tidy settings it produce a redundant type warning